presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -4973,6 +4973,87 @@ public void testShowColumnMetadata()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testRoleGrants()
```

Comment:
update test name to match `role_authorization_descriptors`

Reply:
Done